### DISCUSSION TITLE
Forms as fieldtypes?

### DISCUSSION THREAD

**Discussion by kevadamson** [solspace/craft-express-forms#70] [Feature Requests]
So an entry is created in a section, and then a form can be selected via dropdown ...

**Comment by kevadamson** [comment]
Hi. Any thoughts on this? I need to be able to "attach" a form to an entry ...

**Reply by kjmartens** [reply]
It's noted as a feature request for future consideration. I suppose as a workaround, you could consider having a dropdown field that contains form handles for the values, and pass those off to an expressforms.form() function in your template:

`{% set form = craft.expressforms.form(entry.formFieldinCraftEntry) %}`

**Reply by kevadamson** [reply]
Aha yes. That's a good idea. Plus Solspace does Freeform, right? And that has all the bells and whistles, although this dropdown idea might just do the trick. Thanks Kelsey!

**Reply by kjmartens** [reply]
Yes, Freeform is built by us (Solspace) as well, and definitely has all the bells and whistles, including attaching a form to another Craft element. 😁
